x-pack/test/accessibility/apps/search_profiler.ts

```diff
@@ -14,15 +14,21 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const aceEditor = getService('aceEditor');
   const a11y = getService('a11y');
   const flyout = getService('flyout');
+  const esArchiver = getService('esArchiver');
 
-  // FLAKY: https://github.com/elastic/kibana/issues/91939
-  describe.skip('Accessibility Search Profiler Editor', () => {
+  describe('Accessibility Search Profiler Editor', () => {
     before(async () => {
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
       await PageObjects.common.navigateToApp('searchProfiler');
       await a11y.testAppSnapshot();
       expect(await testSubjects.exists('searchProfilerEditor')).to.be(true);
     });
 
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/logstash_functional');
+      await flyout.ensureAllClosed();
```

Comment:
I think `after()` is run after _all_ the tests have run, correct? If so, I'm not sure this is needed.